quarkus/service/src/main/java/org/apache/polaris/service/quarkus/config/ProductionReadinessChecks.java

```diff
@@ -55,26 +62,53 @@ public class ProductionReadinessChecks {
    */
   private static final String WARNING_SIGN_UTF_8 = "\u0000\u26A0\uFE0F";
 
+  private static final String SEVERE_SIGN_UTF_8 = "\u0000\uD83D\uDED1";
```

Comment:
These changes seem pretty unnecessary

Reply:
I do think that there are nuances in what's a warning and what's severe.

Reply:
That may be so, but I don't think this change is necessary to `Require explict user-consent to enable HadoopFileIO`

Reply:
@eric-maynard I am not sure what the ask is here.  Would you rather have the UTF-8 icon and/or the log message implemented in a separate PR?  Or do you mean that the PR title should be a bit more general?

Reply:
Sure, I think a separate PR for these changes could be a good idea. If this PR really is a bugfix, let's not couple unnecessary changes with it.

Reply:
Let's be mindful on the overhead that too fine-grained PR would cause though...

The changes in this PR are cohesive to one another.  And it is quite small.  So it makes sense to submit the changes in a single PR.  Splitting a PR into many tiny PRs would prevent reviewers from understanding the big picture and would not have any functional merit.

Reply:
It's definitely fair, it's only a question of necessity. If there really is an urgent bugfix to be made, must we include these changes here?